.github/workflows/ai-runner-models.yaml

```diff
@@ -0,0 +1,132 @@
+name: Pull and dockerize huggingface models
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "runner/dl_checkpoints.sh"
+      - ".github/workflows/ai-runner-models.yaml"
+  pull_request:
+    paths:
+      - "runner/dl_checkpoints.sh"
+      - ".github/workflows/ai-runner-models.yaml"
+  workflow_dispatch:
```

Comment:
We just want to redownload/rebuild the models when `dl_checkpoints.sh` changed or on demand.

I think that this should be independent from building ai-runner containers which are based on our code changes.

Reply:
The tricky part is that the `dl_checkpoints.sh` script uses the `ai-runner` image to run the tensorrt step, so there's kind of a cyclic dependency here. But we could have a caveat documented somewhere that one needs to `workflow_dispatch` this manually when the runner image changes in a significant way.